### PR TITLE
CLI Installation

### PR DESCRIPTION
**Pull request by nadinequinr3**

### CODE REVIEW

**Comment by nadinequinr3** [comment]
@ronanbrowne FYI. You might want to check file names. Thanks.

**Review by paulmoloneyr3 — CHANGES_REQUESTED** [review]
Just a tiny one....

File: content/en/platform/corda/5.0-alpha/deploying/installing-corda-cli.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    parent: corda-5-alpha-deploy
+section_menu: corda-5-alpha
+title: Installing the Corda CLI
+---
+Corda CLI (command line interface) is a command line tool that supports various Corda-related tasks, including [Corda Package Installer (CPI)](../introduction/key-concepts.html#corda-package-installer-cpi) creation and Corda cluster management.
+
+## Downloading Corda CLI
+
+To obtain the Corda CLI installer:
+1. Download `platform-jars-Eagle.tar.gz` from the [R3 Customer Hub](https://r3.force.com/)
```
Comment (colin-hillis) [inline content/en/platform/corda/5.0-alpha/deploying/installing-corda-cli.md:16]: period missing

File: content/en/platform/corda/5.0-alpha/deploying/installing-corda-cli.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+   ```
+   The script installs Corda CLI to `<user-home>/.corda/cli`, where `<user-home>` refers to your user home directory. For example, on macOS, this is typically something like `/Users/charlie.smith` or on Linux, something like `/home/charlie.smith`.
+
+## Installing on Windows
+
+1. Start a Powershell session.
+2. Change directory to where you saved `corda-cli-downloader-Eagle.zip`.
+3. Extract the contents of the `zip` file:
+   ```shell
+   Expand-archive .\corda-cli-downloader-Eagle.zip
```
Comment (colin-hillis) [inline content/en/platform/corda/5.0-alpha/deploying/installing-corda-cli.md:43]: I already asked about the capital e in Extract, but should eagle be capitalised too? I assume so, based on below instance, but just checking.

Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0-alpha/deploying/installing-corda-cli.md:43]:
```suggestion
   Expand-Archive .\corda-cli-downloader-Eagle.zip
```

**Review by paulmoloneyr3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.0-alpha/deploying/installing-corda-cli.md (hunk 2 above)
Comment (nadinequinr3) [reply content/en/platform/corda/5.0-alpha/deploying/installing-corda-cli.md]: Based on this https://r3-cev.atlassian.net/wiki/spaces/CB/pages/4254793729/C5+Beta+Program+-+Alpha+Release+-+RC05+Code+name+Eagle it should but I've asked @ronanbrowne to confirm

Comment (nadinequinr3) [reply content/en/platform/corda/5.0-alpha/deploying/installing-corda-cli.md]: Confirmed - capital "e" for Eagle.
(It's generated from here https://github.com/corda/corda-runtime-os/blob/release/os/5.0-Alpha/gradle.properties#L43)

**Review by colin-hillis — APPROVED** [review]

**Review by paulmoloneyr3 — APPROVED** [review]

**Pull request merged**